metadata.json

```diff
@@ -4,7 +4,9 @@
   "description": "Organize windows in customizable snap zones like FancyZones on windows.",
   "version": 1.1,
   "shell-version": [
-    "40"
+    "40",
+    "41",
+    "41.1"
```

Comment:
Just "41" works even on Gnome 41.1.

Reply:
I was doubting that. First time doing this. Pushed another commit.